include/fastdds/statistics/monitorservice_types.idl

```diff
@@ -0,0 +1,131 @@
+// Copyright 2023 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @file include/fastdds/statistics/ms_types.idl
+ */
+
+#include "types.idl"
+
+module eprosima {
+module fastdds {
+module statistics {
+
+	/* Reuse existing statistics detail module*/
+	module detail
+	{
+		typedef eprosima::fastdds::statistics::detail::EntityId_s EntityId_m;
+		typedef eprosima::fastdds::statistics::detail::GUID_s GUID_m;
+		typedef eprosima::fastdds::statistics::detail::GuidPrefix_s GuidPrefix_m;
+		typedef eprosima::fastdds::statistics::detail::Locator_s Locator_m;
+	};
```

Comment:
Does it make sense to define the alias when with the latest refactor these types are defined in the same namespace?